docs/quick-start.md

```diff
@@ -0,0 +1,94 @@
+
+This guide assumes you are using MacOS or a Linux machine and are an Automattician. For more detailed information, including alternate local dev environments, running unit tests, best coding practices, and more, you can use the [full Development Environment guide here](development-environment.md#clone-the-repository). 
+
+## Installation
+
+Prior to installation, we recommend using [`Homebrew`](https://brew.sh/) to manage installations and [`nvm`](https://github.com/nvm-sh/nvm/) to manage Node.js versions. If you don't already have those installed, you can do so by copy/pasting each of the following commands and running them in your terminal:
+
+- Homebrew: `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+- nvm: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash`
+
+The Jetpack Monorepo requires the following to be installed on your machine:
+
+- Start by cloning the GitHub repo:
+	- `git clone https://github.com/Automattic/jetpack.git`
+	- If you're not an Automattician, you can [fork the repo following the instructions here](https://docs.github.com/en/get-started/quickstart/contributing-to-projects).
+- Bash (will need to be updated from default Mac version): `brew install bash`
+- jq (JSON processor used in scripts): `brew install jq` 
+- Node.js (used for build process and our CLI): `nvm install && nvm use`
+- PNPM (a Node.js package manager): `npm install -g pnpm`
+- Composer (our PHP package manager): `brew install composer`
+- PHP (the language at the core of the WordPress ecosystem): `source .github/versions.sh && brew install php@$PHP_VERSION`
+- Jetpack CLI (an internal tool that assists with development): `pnpm install && pnpm cli-setup`
+	- [You can read more about using the CLI here](https://github.com/Automattic/jetpack/blob/master/tools/cli/README.md).
+
+## Running Jetpack locally
+
+After everything is installed, you're ready to run Jetpack locally! While there are other supported methods of doing this, we recommend and support using Docker containers. 
+
+To setup Docker:
+- Install Docker:
+	- Mac: `brew install --cask docker`
+	- Linux: `brew install docker` 
+- Copy the settings file and modify the site password and any other configuration you'd like: `cp tools/docker/default.env tools/docker/.env`
+- Start the Docker container using `jetpack docker up`
+- Install WordPress in your Docker container using `jetpack docker install` 
+- Open up http://localhost to see your site!
+
+For more in depth Docker instructions, follow the [Docker environment for Jetpack Development guide](../tools/docker/README.md).
+
+### Setting up Jurassic Tube
+
+To test network related features of Jetpack, you'll need a test site that can create local HTTP tunnels. If you're an Automattician, we recommend using Jurassic Tube:
+
+- Add a subdomain on [jurassic.tube](https://jurassic.tube/)
```

Comment:
Does this subdomain need to get added into the .env file for Docker? Or does just leaving it as `localhost` still allow it to work? This is one of those parts where I look at the .env file and am a bit uncertain whether to change things or not because I feel like it hasn't been optimized for JT yet.

Reply:
Nope, you can leave the .env file alone for Jurassic Tube.

Reply:
cool thanks. i'm just misremembering i guess.